File: c#/program16.cs
using System;

class program16
{
    public static int CountEven(int No)
    {
        int iDigit = 0,count = 0;

        while(No != 0)
        {
            iDigit = No % 10;
            if((iDigit%2) == 0)
            {
                count++;
            }
            No = No / 10;
        }
        return count;
    }
    
    public static void Main(string[] args)
    {
        int value = 0, ret = 0;

        Console.WriteLine("Enter number :");
        value = int.Parse(Console.ReadLine());

        ret = CountEven(value);
        Console.WriteLine("Count of the even digit is : {0}",ret);
    }
}

File: java/program16.java
import java.util.*;

class Digit
{
    public int iNo;

    public Digit(int X)
    {
        iNo = X;
    }

    public int CountEven()
    {
        int iDigit = 0;
        int iCount = 0;
        int temp = iNo;

        while(temp != 0)
        {
            iDigit = temp % 10;
            if((iDigit % 2) == 0)
            {
                iCount++;
            }
            temp = temp / 10;
        }
        return iCount;
    }
}

class program16
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number ");
        int iValue = sobj.nextInt();

        Digit obj = new Digit(iValue);

        System.out.println("Count of the Even digits is : "+obj.CountEven());
        System.out.println("Count of the Even digits is : "+obj.CountEven());
    }

}